apps/analytics/views.py

```diff
@@ -118,7 +118,7 @@ def get_challenge_phase_submission_analysis(request, challenge_pk, challenge_pha
     challenge_phase = get_challenge_phase_model(challenge_phase_pk)
 
     submissions = Submission.objects.filter(
-        challenge_phase=challenge_phase, challenge_phase__challenge=challenge)
+        challenge_phase=challenge_phase, challenge_phase__challenge=challenge, created_by=request.user.pk)
```

Comment:
This check has to be done at the participant team level and not at the level of `created_by`. Please change this.

Reply:
Also, please add test cases for this.